Source/Scene/PrimitivePipeline.js

```diff
@@ -45,6 +47,11 @@ define([
             var modelMatrix = instances[0].modelMatrix;
 
             for (i = 1; i < length; ++i) {
+                var offset = instances[i].attributes.offset;
+                if (defined(offset)) {
+                    toWorld = true;
```

Comment:
This will break things that depend of changing the model matrix, like boxes and ellipsoids.